packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -172,6 +173,7 @@ class ThemeData extends Diagnosticable {
     // Spec doesn't specify a dark theme secondaryHeaderColor, this is a guess.
     secondaryHeaderColor ??= isDark ? Colors.grey[700] : primarySwatch[50];
     textSelectionColor ??= isDark ? accentColor : primarySwatch[200];
+    cursorColor = const Color.fromRGBO(66, 133, 244, 1.0);
```

Comment:
Isn't this supposed to be derived from the primary swatch? @willlarche

Reply:
I will be talking to Will about this tomorrow